CONTINUOUS_INTEGRATION.md

```diff
@@ -0,0 +1,198 @@
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 numpy_vers/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 package/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 package-tensorflow/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 python-build/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 run-tests/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 select-xcode/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 setup-tensorflow/
+4 drwxrwxr-x  2 root root 4096 Apr 20 22:52 win-install-sox/
+```
+
+Within each of these directories is a `.yml` file that defines an _Action_. Each _Action_ calls a `bash` script. The `bash` scripts are held in the `ci_scripts` directory in the `DeepSpeech` repository:
+
+```
+~/DeepSpeech/ci_scripts$ ls
+total 164
+ 4 drwxrwxr-x  2 root root  4096 Apr 20 22:52 ./
+ 4 drwxrwxr-x 16 root root  4096 Apr 20 22:52 ../
+ 4 -rwxrwxr-x  1 root root  3678 Apr 20 22:52 all-utils.sh
+ 4 -rwxrwxr-x  1 root root  2472 Apr 20 22:52 all-vars.sh
+24 -rwxrwxr-x  1 root root 23668 Apr 20 22:52 asserts.sh
+ 4 -rwxrwxr-x  1 root root  1300 Apr 20 22:52 build-utils.sh
+ 4 -rwxrwxr-x  1 root root   382 Apr 20 22:52 cpp-bytes-tests.sh
+ 4 -rwxrwxr-x  1 root root   488 Apr 20 22:52 cpp-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   345 Apr 20 22:52 cpp-tests.sh
+ 4 -rwxrwxr-x  1 root root   428 Apr 20 22:52 cpp_tflite_basic-tests.sh
+ 4 -rwxrwxr-x  1 root root   569 Apr 20 22:52 cpp_tflite-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   541 Apr 20 22:52 cpp_tflite-tests.sh
+ 4 -rwxrwxr-x  1 root root   318 Apr 20 22:52 cppwin-tests.sh
+ 4 -rwxrwxr-x  1 root root   467 Apr 20 22:52 cppwin_tflite-tests.sh
+ 4 -rw-rw-r--  1 root root   316 Apr 20 22:52 docs-requirements.txt
+ 4 -rwxrwxr-x  1 root root   602 Apr 20 22:52 electronjs-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   384 Apr 20 22:52 electronjs-tests.sh
+ 4 -rwxrwxr-x  1 root root   631 Apr 20 22:52 electronjs_tflite-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   523 Apr 20 22:52 electronjs_tflite-tests.sh
+ 4 -rwxrwxr-x  1 root root   538 Apr 20 22:52 host-build.sh
+ 4 -rwxrwxr-x  1 root root   556 Apr 20 22:52 node-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   343 Apr 20 22:52 node-tests.sh
+ 4 -rwxrwxr-x  1 root root   591 Apr 20 22:52 node_tflite-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   482 Apr 20 22:52 node_tflite-tests.sh
+ 4 -rwxrwxr-x  1 root root   614 Apr 20 22:52 package.sh
+ 4 -rwxrwxr-x  1 root root  2787 Apr 20 22:52 package-utils.sh
+ 4 -rwxrwxr-x  1 root root   521 Apr 20 22:52 python-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   274 Apr 20 22:52 python-tests.sh
+ 4 -rwxrwxr-x  1 root root   502 Apr 20 22:52 python_tflite-tests-prod.sh
+ 4 -rwxrwxr-x  1 root root   413 Apr 20 22:52 python_tflite-tests.sh
+ 4 -rwxrwxr-x  1 root root  2688 Apr 20 22:52 tf-build.sh
+ 4 -rwxrwxr-x  1 root root  2384 Apr 20 22:52 tf-package.sh
+ 4 -rwxrwxr-x  1 root root  3371 Apr 20 22:52 tf-setup.sh
+12 -rwxrwxr-x  1 root root  8375 Apr 20 22:52 tf-vars.sh
+```
+
+### Runners
```

Comment:
I'm unsure whether we should ever cover this. All of the work we did was ensuring this can run using GitHub Actions provided runners, and not asking people to spawn and maintain their own hardware (which was something we did on TaskCluster to address specific needs, and is a lot of work).

Reply:
Right, I'd just defer to GitHub documentation.

Reply:
Good call, I have removed this reference.